azavea/tilejson.io
# Add ability to use same zoom and center on shares and diffs

## Overview

Currently the zoom on shares and on diff mode is always 2 and center is always [0,0]. This change enables saving zoom and center state and using it in diffs and shares. 

Connects #45 

## Testing Instructions

 * Add layers
 * Zoom in and change center
 * Share the map and check if zoom and center are the same
 * Go into diff mode and check if zoom and center are the same
 * Change zoom and center in diff mode
 * Share the diff and check if zoom and center are the same
